pkg/template/static_context.go

```diff
@@ -390,14 +395,99 @@ func (ctx StaticCtx) tlsKey(certName string, args ...interface{}) string {
 	return p.Key
 }
 
+func (ctx StaticCtx) tlsCaCert(caName string, args ...interface{}) string {
```

Comment:
any reason not to make the args explicit here? i believe the reason tlsKey uses arbitrary args was to preserve backwards compatibility

https://github.com/replicatedhq/kots/commit/38898458f0cbc5bc2b3b785b5a7b2520a9a7f7dd#diff-b82706ecb35ffdea1e9f78aa454d2ec8R348

Reply:
Ah, wasn't sure why that choice was made. It seemed reasonable to allow calling TLSCertFromCA then call TLSCACert with just the CA identifier to get the cert that had already been generated.